Unable to start Cloudflare workers project with capital letters in directory name

This may be a minor issue, but I'll share.

I ran the following

```
$ npm create hono@latest todoAPI
Need to install the following packages:
create-hono@0.3.1
Ok to proceed? (y) y

create-hono version 0.3.1
✔ Using target directory … todoAPI
✔ Which template do you want to use? › cloudflare-workers
cloned honojs/starter#main to /workspaces/*****/todoAPI
✔ Copied project files
```

After `cd todoAPI && npm i` , Run the following

```
$ npm run dev

> dev
> wrangler dev src/index.ts

 ⛅️ wrangler 3.14.0
-------------------

✘ [ERROR] Processing wrangler.toml configuration:

    - Expected "name" to be of type string, alphanumeric and lowercase with dashes only but got
  "todoAPI".
```

The wrangler.toml generated at this time is as follows

```toml:wrangler.toml 
name = "todoAPI"
compatibility_date = "2023-01-01"

.......
```

I thought it would be better to kebab case and convert all to lower case when replacing by project name from the template.
https://github.com/honojs/create-hono/blob/6aa68a3be01d4e1ed5034b5701ce83017e7f731e/src/hooks/after-create.ts#L12

However, I can't use this method when the project name is a non-ascii character or a non-hyphen symbol. Therefore, I think it would be better to dare to do nothing.

--- a/src/hooks/after-create.test.ts
+++ b/src/hooks/after-create.test.ts
@@ -21,14 +21,14 @@ name = "%%PROJECT_NAME%%-staging"
         })
         const { readFileSync, writeFileSync } = await import('fs')
 
-        const projectName = 'test-project'
+        const projectName = 'test-projectNAME+123'
         const directoryPath = './tmp'
         const wranglerPath = join(directoryPath, 'wrangler.toml')
         const replaced = `
-name = "${projectName}"
+name = "test-projectname-123"
 
 [env.staging]
-name = "${projectName}-staging"
+name = "test-projectname-123-staging"
     `.trim()
         afterCreateHook.applyHook('cloudflare-workers', {
           projectName,

--- a/src/hooks/after-create.ts
+++ b/src/hooks/after-create.ts
@@ -9,7 +9,8 @@ afterCreateHook.addHook(
   ({ projectName, directoryPath }) => {
     const wranglerPath = path.join(directoryPath, 'wrangler.toml')
     const wrangler = readFileSync(wranglerPath, 'utf-8')
-    const rewritten = wrangler.replaceAll(PROJECT_NAME, projectName)
+    const convertProjectName = projectName.toLowerCase().replaceAll(/[^a-z0-9\-_]/gm, "-")
+    const rewritten = wrangler.replaceAll(PROJECT_NAME, convertProjectName)
     writeFileSync(wranglerPath, rewritten)
   }
 )

